packages/preview/hhn-unitylab-thesis-template/0.0.2/template/10-Pre-Doc/10-Pre-Doc.typ

```diff
@@ -0,0 +1,12 @@
+#import "../../src/config.typ": *
+//#import "@preview/unitylab-thesis-template:0.0.1": *
+
```

Comment:
Comment for all your template files, not just this one: the first import should be removed, and the second one un-commented (and the version updated to 0.0.2). I also think it would be better to only import the functions you actually use in the file and not everything (and to not have it at all if not necessary).